pytest_asyncio/plugin.py

```diff
@@ -24,7 +25,11 @@
 )
 
 import pytest
-from typing_extensions import Literal
+
+if sys.version_info >= (3, 8):
```

Comment:
I can change this to use `sys.hexversion` or `ImportError` approach if you prefer.